server.js

```diff
@@ -158,14 +158,25 @@ router.get('/blog/2020-04-21-data-insights', (req, res) => {
 router.get(['/whoweare', '/whoweare.html'], (req, res) => {
   res.render('whoweare', {
     layout: 'static',
-    ogLocale:  formatFbLocale(res.locals.locale),
+    ogLocale: formatFbLocale(res.locals.locale),
     ogTitle: res.locals.banana.i18n('ftm-about-us-og-title'),
     ogUrl: `http://${req.hostname}${req.originalUrl}`,
     ogDescription: res.locals.banana.i18n('ftm-default-og-description'),
     version: herokuVersion
   });
 });
 
+router.get(['/partners', '/partners.html'], (req, res) => {
```

Comment:
nit: we don't need the `/partners.html` here. The give/whoweare/index are legacy for backwards compatibility from before this was an express server